changelog/upgrade-notes.md

```diff
@@ -33,6 +33,10 @@ Users compiling their code with the library should note that the following typed
 * ol.style.RegularShapeRenderOptions to ol.RegularShapeRenderOptions
 * ol.style.StyleFunction to ol.StyleFunction
 
+#### KML format now uses URL()
+
+The URL constructor is supported by all modern browsers, but not by older ones, such as IE. To use the KML format in such older browsers, a URL polyfill will have to be loaded before use.
```

Comment:
But isn't it also translating a relative url into an absolute url in the case of the KML format?